packages/coinstac-ui/app/main/utils/boot/configure-core.js

```diff
@@ -6,12 +6,16 @@ const CoinstacClientCore = require('coinstac-client-core');
 const { merge } = require('lodash');
 const parseCLIInput = require('./parse-cli-input.js');
 const PouchDBAdapterLevelDB = require('pouchdb-adapter-leveldb');
+const PouchDBAdapterMemory = require('pouchdb-adapter-memory');
 const url = require('url');
 
 // TODO: Yeeikes
 common.services.dbRegistry.DBRegistry.Pouchy.PouchDB.plugin(
   PouchDBAdapterLevelDB
 );
+common.services.dbRegistry.DBRegistry.Pouchy.PouchDB.plugin(
```

Comment:
Gotta add the memory adapter somewhere. Might as well be here to keep plugin addition in one location.